createljspeech.py

```diff
@@ -0,0 +1,54 @@
+# This script generates a folder structure for ljspeech-1.1 processing from mimic-recording-studio database
+# Written as first python script (eg. helloworld) by Thorsten Mueller (MrThorstenM@gmx.net) on november 2019 without any warranty
+
+import sqlite3
+import os
+from datetime import datetime
+from shutil import copyfile
+
+# please set following vars matching your environment
+# ---------------------------------------------------
+
+# Directory of mimic-recording-studio (eg: /home/thorsten/mimic-recording-studio)
+directory_base_mrs = "/home/thorsten/tts/mimic-recording-studio"
+# Output directory for ljspeech structure (eg: /tmp/tts)
+directory_base_ljspeech = "/home/thorsten/tts"
```

Comment:
Should this be set to the `tacotron` directory then users won't need to copy it across?

Reply:
Also could use [`os.path.expanduser`](https://stackoverflow.com/questions/4028904/how-to-get-the-home-directory-in-python#4028943) to generalize these paths

Reply:
Thanks for these hints. I will modify the script appropriate.